src/daria/utils/box.py

```diff
@@ -62,3 +62,29 @@ def bounding_box_inverse(bounding_box: tuple) -> np.ndarray:
     )
 
     return coords
+
+
+def perimeter(box: Union[tuple, np.ndarray]) -> Union[int, float]:
+    """
+    Returns the perimeter of a box. Accepts both tuples of slices
+    as well as arrays of coordinates as input.
+
+    Args:
+        box (tuple of slices or np.ndarray): definition of a box,
+            either as tuple of slices, or coordinates (can also
+            use metric units)
+
+    Returns:
+        float or int (depending on input): perimeter
+    """
+    if isinstance(box, np.ndarray):
+        min_x = np.min(box[:, 0])
+        max_x = np.max(box[:, 0])
+        min_y = np.min(box[:, 1])
+        max_y = np.max(box[:, 1])
+        perimeter = 2 * (max_x - min_x) + 2 * (max_y - min_y)
+
+    elif isinstance(box, tuple):
+        raise NotImplementedError("This is currently not supported.")
```

Comment:
```suggestion
       box_array =  bounding_box_inverse(box)
       min_x = np.min(box_array[:, 0])
       max_x = np.max(box_array[:, 0])
       min_y = np.min(box_array[:, 1])
       max_y = np.max(box_array[:, 1])
       perimeter = 2 * (max_x - min_x) + 2 * (max_y - min_y)
       
```

Reply:
I just wrote this in github now, not 100% sure if it is exactly what you want. Just disregard it if that is the case.

Reply:
I took the libert to disregard this suggestion due to code duplication. Instead, unified conversion is provided at the start of the routine.